src/include/model_server/model_server_manager.h

```diff
@@ -0,0 +1,127 @@
+#pragma once
+
+#include <filesystem>
+#include <string>
+#include "loggers/messenger_logger.h"
+#include "messenger/messenger.h"
+#include "messenger/messenger_defs.h"
+
+namespace noisepage::model {
+static constexpr const char *MODEL_CONN_ID_NAME = "model-server-conn";
+static constexpr const char *MODEL_TARGET_NAME = "model";
+static constexpr const char *MODEL_IPC_PATH = "/tmp/model-server-ipc";
+static constexpr const char *MODEL_TCP_HOST = "127.0.0.1";
+static constexpr const int MODEL_TCP_PORT = 15645;
```

Comment:
this is more of a note to me, but we need to start a definitive "these are the ports we are using"